code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -435,8 +435,8 @@
 	if(!beaker || (beaker && beaker.reagents.holder_full()))
 		return
 	playsound(src.loc, 'sound/machines/blender.ogg', 50, 1)
```

Comment:
```suggestion
	playsound(loc, 'sound/machines/blender.ogg', 50, 1)
```
while you're here

Reply:
`1` Should also be replaced with `TRUE`, while we are here.